source/declarations.tex

```diff
@@ -5809,11 +5809,11 @@
 \end{example}
 
 \item Otherwise, if \tcode{T} is a reference type, a prvalue is generated.
-The prvalue initializes its result object by copy-list-initialization.
+The prvalue initializes its result object by copy-list-initialization from the initializer list.
```

Comment:
@jensmaurer Do you prefer "initializes with" or "initializes from"?

Reply:
"from" sounds fine here.